dex/router/src/contract.rs

```diff
@@ -1,5 +1,4 @@
 #![no_std]
-#![feature(generic_associated_types)]
 
```

Comment:
there is a clippy warning from past implementation

Reply:
done